src/DIPS.Xamarin.UI/Controls/FloatingActionMenu/MenuButton.xaml.cs

```diff
@@ -326,10 +326,26 @@ public string Title
             set => SetValue(TitleProperty, value);
         }
 
+        public new static readonly BindableProperty IsVisibleProperty = BindableProperty.Create(nameof(IsVisible), typeof(bool), typeof(MenuButton), propertyChanged:OnIsVisibleChanged, defaultBindingMode: BindingMode.TwoWay);
+
+        private static void OnIsVisibleChanged(BindableObject bindable, object oldvalue, object newvalue)
+        {
+            if (bindable is MenuButton menuButton)
+            {
+                menuButton.FloatingActionMenuParent?.UpdateMenuButtonVisibility(menuButton, (bool)newvalue);
+            }
+        }
+
+        public new bool IsVisible
```

Comment:
Missing comment, just use inheritdoc if it's possible

Reply:
Added